app/components/AddTasks.tsx

```diff
@@ -31,6 +31,13 @@ const AddTasks = () => {
         className="flex-grow bg-transparent outline-none placeholder-gray-500"
         placeholder="Add a task..."
       />
+      {/** TODO: build a Select to choose the task color - string selections like 'red' 'blue' etc */}
+      {/* <select>
+        <option value="red">Red</option>
+        <option value="blue">Blue</option>
+        <option value="green">Green</option>
+      </select> */}
+
```

Comment:
I want to build a set number of selections for color, and map them to different hues of that color based on the theme the user has selected.